src/ProductConstructionService/ProductConstructionService.Api/VMRConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace ProductConstructionService.Api;
+
+public static class VMRConfiguration
+{
+    public const string VmrPathKey = "VmrPath";
+    public const string TmpPathKey = "TmpPath";
+
+    public static void AddVmrRegistrations(this WebApplicationBuilder builder)
+    {
+        builder.Services.TryAddSingleton<IBasicBarClient>(sp => sp.GetRequiredService<IBarApiClient>());
```

Comment:
This is not correct. We need to give it the `SqlBarClient` implementation. This one would call for data over HTTP to Maestro.

Reply:
fixed